### DISCUSSION TITLE
How send email from Laravel?

### DISCUSSION THREAD

**Discussion by mariuszmalek** [brefphp/bref#944] [Support] [answered]
I have that error when I try to send any of email (by worker)

View [emails.auth.reset.begin] not found. {"exception":"[object] (InvalidArgumentException(code: 0): View [emails.auth.reset.begin] not found. at /var/task/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137) | [2021-06-04 10:50:20] production.ERROR: View [emails.auth.reset.begin] not found. {"exception":"[object] (InvalidArgumentException(code: 0): View [emails.auth.reset.begin] not found. at /var/task/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137)
-- | --

**Comment by mnapoli** [comment, marked as answer]
Hey, it seems like a Laravel error. Are you sure that the view exists?

**Reply by mariuszmalek** [reply]
Everything works fine only in localhost. Already I don't know what is happening.
